From: Jan Engelhardt <jengelh@inai.de>
Date: 2016-06-30 13:50:19.632475047 +0200

The documentation has the phrase

"The API is not yet 100% stable, so we do not yet guarantee ABI
forward compatibility. We will do so, once we increase the shared
library version above 1.0."

[Wrong premise. On the technical level, API/ABI stability does not
need to be guaranteed at all, nor is there a requirement that it has
to happen/start at a particular time, nor at a particular version
like "1.0".

What you MUST do is that *if* the ABI/API changes, change
the SONAME.]

Add -release to address
https://en.opensuse.org/openSUSE:Shared_library_packaging_policy#When_there_is_no_versioning
point 3. [until libsass 1.0, then reevaluate]

---
 src/GNUmakefile.am |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: libsass-3.3.2/src/GNUmakefile.am
===================================================================
--- libsass-3.3.2.orig/src/GNUmakefile.am
+++ libsass-3.3.2/src/GNUmakefile.am
@@ -34,7 +34,7 @@ include $(top_srcdir)/Makefile.conf
 
 libsass_la_SOURCES = ${CSOURCES} ${SOURCES}
 
-libsass_la_LDFLAGS = $(AM_LDFLAGS) -no-undefined -version-info 1:0:0
+libsass_la_LDFLAGS = $(AM_LDFLAGS) -no-undefined -version-info 1:0:0 -release ${PACKAGE_VERSION}
 
 if ENABLE_TESTS
 if ENABLE_COVERAGE
